electronic_store/demo/stock_lot.xml

```diff
@@ -0,0 +1,173 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="stock_lot_1" model="stock.lot">
+        <field name="name">804KCPY43321</field>
+        <field name="product_id" ref="product_product_1037"/>
+        <field name="location_id" ref="stock.stock_location_customers"/>
```

Comment:
Maybe remove location_id for all
```suggestion
```

Reply:
Are locations useful for others stock lots? :thinking: You can remove all in my opinion